plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/ui/AppEngineStandardProjectConvertCommandHandlerTest.java

```diff
@@ -113,6 +115,8 @@ public void testCheckFacetCompatibility_java1_6FacetIsIncompatible() throws Core
 
   @Test
   public void testCheckFacetCompatibility_java1_8FacetIsIncompatible() throws CoreException {
+    Assume.assumeTrue("AppEngine Standard Java 8 runtime support is not present",
```

Comment:
It's testing that when our AESv1 facet claims to not support Java 8, that our convert command properly reports that it cannot convert the project.  It's no different from a java8 feature flag and having the `Assume.assumeFalse(java8)`.